en/PlanView/PlanView.md

```diff
@@ -11,33 +11,39 @@ The image above shows a simple mission which starts with a takeoff, flies throug
 The steps to creating a missions are:
 
 1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
+2. Add waypoints to the mission and edit as needed
```

Comment:
Is the terminology really waypoint? I an action to take a camera picture or change speed as a command or a mission item - not a waypoint.

Reply:
I didn't consider other commands. How about "Add waypoints or commands" ?

Reply:
That works for me :-)